.github/workflows/main-branch-build.yml

```diff
@@ -0,0 +1,32 @@
+# This workflow will do a clean installation of node dependencies, cache/restore them, build the source code and run tests across different versions of node
+# For more information see: https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-nodejs
+
+
+name: Event Pro CI

```

Comment:
If we are building for prod, we can rename this to "Build for Production" or something like that